incubator/tensorflow-notebook/templates/secrets.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "tensorflow-notebook.fullname" . }}
+  labels:
+    app: {{ template "tensorflow-notebook.name" . }}
+    chart: {{ template "tensorflow-notebook.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
```

Comment:
I wonder what this hook is needed for.

Reply:
This hook means secrets should be ready before other resources load into kubernetes.